src/test/regress/expected/multi_tenant_isolation_nonblocking.out

```diff
@@ -497,20 +497,26 @@ SET citus.override_table_visibility TO false;
       Schema      |            Name            | Type  |         Owner
 ---------------------------------------------------------------------
  Tenant Isolation | lineitem_streaming         | table | mx_isolation_role_ent
+ Tenant Isolation | lineitem_streaming_1230000 | table | mx_isolation_role_ent
```

Comment:
would be good to add appropriate cleanup calls to not change the intent of these tests

Reply:
@marcocitus , Since tenant isolation will run with deferred drop on by default, I was expecting to keep these changes. Is flakiness the concern here?

Reply:
Fixed.